examples/flutter_gallery/test_driver/transitions_perf.dart

```diff
@@ -4,16 +4,27 @@
 
 import 'dart:async';
 import 'dart:convert' show JsonEncoder;
+import 'dart:ui' show window, Size;
 
 import 'package:flutter_driver/driver_extension.dart';
 import 'package:flutter_gallery/gallery/demos.dart';
 import 'package:flutter_gallery/main.dart' as app;
 
 Future<String> _handleMessages(String message) async {
-  assert(message == 'demoNames');
-  return const JsonEncoder.withIndent('  ').convert(
-    kAllGalleryDemos.map((GalleryDemo demo) => '${demo.title}@${demo.category.name}').toList(),
-  );
+  assert(message == 'demoNames' || message == 'mediaSize');
+  switch(message) {
+    case 'demoNames':
+      return const JsonEncoder.withIndent('  ').convert(
+        kAllGalleryDemos.map((GalleryDemo demo) => '${demo.title}@${demo.category.name}').toList(),
+      );
+    case 'mediaSize':
+      final Size size = window.physicalSize / window.devicePixelRatio;
+      return const JsonEncoder.withIndent('  ').convert(<String, double>{
+        'width': size.width,
+        'height': size.height,
+      });
+  }
+  return '';
```

Comment:
We probably want to fail in this case.

Reply:
Right. Given the asserts at the top of the function, I think ` assert(false, 'not reachable');` would do here.

Reply:
Added the assert(). Still stuck with the return to satisfy the analyzer.